tests/FSharp.Compiler.ComponentTests/CompilerCompatibilityTests.fs

```diff
@@ -0,0 +1,113 @@
+module FSharp.Compiler.ComponentTests.CompilerCompatibilityTests
+
+open System
+open System.IO
+open Xunit
+open FSharp.Test
+open FSharp.Test.Assert
+open TestFramework
+
+type CompilerCompatibilityTests() =
+
+    let projectsPath = Path.GetFullPath(Path.Combine(__SOURCE_DIRECTORY__, "../projects/CompilerCompat"))
+    let libProjectPath = Path.Combine(projectsPath, "CompilerCompatLib")
+    let appProjectPath = Path.Combine(projectsPath, "CompilerCompatApp")
+    
+    let runDotnetBuild projectPath useLocalCompiler =
+        let args = 
+            if useLocalCompiler then
+                "build -c Release -p:LoadLocalFSharpBuild=True"
```

Comment:
How does this actually do anything? The `.fsproj` would need to be aware of this property and do something based on it.